sdk/keyvault/keyvault-certificates/tsconfig.eslint.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
what is this file used for?

Reply:
ESLint now complains since our samples don't exist in our tsconfig.json. This file sole purpose is for ESLint to acknowledge that the samples exist, without having to change the main tsconfig.json file.